Orange/widgets/data/owfile.py

```diff
@@ -179,6 +185,30 @@ def _url_set(self):
         self.source = self.URL
         self.load_data()
 
+    #Filling comboBox With Sheet Names
```

Comment:
These types of comments are wrong. Wherever you've seen them, forget it. If anything, use docstrings.